src/robusta/integrations/mail/sender.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import tempfile
+from typing import List
+
+import apprise
+from apprise import NotifyFormat, NotifyType
+
+from robusta.core.reporting.base import BaseBlock, Emojis, Finding, FindingStatus
+from robusta.core.reporting.blocks import (
+    FileBlock,
+    LinksBlock,
+    LinkProp,
+    MarkdownBlock,
+)
+from robusta.core.reporting.consts import EnrichmentAnnotation
+from robusta.core.sinks.transformer import Transformer
+
+
+def with_attr(obj, attr_name, attr_value):
+    setattr(obj, attr_name, attr_value)
+    return obj
+
+
+class MailTransformer(Transformer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.file_blocks = []
+
+    def block_to_html(self, block: BaseBlock) -> str:
+        # TODO should we additionally support ScanReportBlock here?
+        if isinstance(block, FileBlock):
+            self.file_blocks.append(block)
+            return f"<p>See attachment {block.filename}</p>"
+        elif isinstance(block, LinksBlock):
+            if getattr(block, "html_class", None):
+                class_part = f' class="{block.html_class}"'
+            else:
+                class_part = ""
+            return (
+                f"<ul{class_part}>\n"
+                + "\n".join(f'  <li><a href="{link.url}">{link.text}</a></li>' for link in block.links)
+                + "\n</ul>\n"
+            )
+        else:
+            return super().block_to_html(block)
+
+
+class MailSender:
+    def __init__(self, mailto: str, account_id: str, cluster_name: str, signing_key: str):
+        self.mailto = mailto
+        self.signing_key = signing_key
+        self.account_id = account_id
+        self.cluster_name = cluster_name
+
+    def send_finding_via_email(self, finding: Finding, platform_enabled: bool):
+        blocks: List[BaseBlock] = []
+
+        if finding.title:
+            status: FindingStatus = (
+                FindingStatus.RESOLVED if finding.title.startswith("[RESOLVED]") else FindingStatus.FIRING
+            )
+            blocks.append(self.__create_finding_header(finding, status))
+        else:
+            # TODO is this correct? Is it possible for the finding to have no title at all?
```

Comment:
I think it's not possible, and `title` is mandatory

Reply:
removed the "else" part